php-checks/src/main/java/org/sonar/php/checks/security/XxeCheck.java

```diff
@@ -129,10 +129,13 @@ private PreciseIssue createIssue(Tree tree) {
   }
 
   private static String namespaceMemberFullQualifiedName(ExpressionTree callee) {
-    return ((ClassNamespaceNameTreeImpl) ((MemberAccessTreeImpl) callee).object())
-      .symbol()
-      .qualifiedName()
-      .toString();
+    return Optional.of(callee)
+      .filter(MemberAccessTreeImpl.class::isInstance)
+      .map(c -> ((MemberAccessTreeImpl) c).object())
+      .filter(ClassNamespaceNameTreeImpl.class::isInstance)
+      .map(ClassNamespaceNameTreeImpl.class::cast)
+      .map(c -> c.symbol().qualifiedName().toString())
+      .orElse("");
```

Comment:
I would not return an empty String, I would return an `Optional<String>` which you can filter and check by `ifPresent()`